### PR TITLE
SDK-512 Medium severity vulnerability. 'Information Exposure' in module 'follow-redirects'

### PR DESCRIPTION
**Pull request by igor-alekseev-dev**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-512](https://incountry.atlassian.net/browse/SDK-512)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

**Review by artemgolozubov — APPROVED** [review]

**Review by RommeroInCountry — APPROVED** [review]

File: tests/integration/find-records-tests.ts [hunk 1]
```diff
@@ -410,7 +410,7 @@ describe('Find records', () => {
       context('Records search by searchKeys', () => {
         let searchableStorage: Storage;
         const record = createRecordData();
-        record.key2 = '蘋果筆';
+        record.key2 = 'aaaabbbbcccc';
```
Comment (artemgolozubov) [inline tests/integration/find-records-tests.ts:413]:
```suggestion
```

File: tests/integration/find-records-tests.ts [hunk 2]
```diff
@@ -419,7 +419,7 @@ describe('Find records', () => {
             hashSearchKeys: false,
           });
 
-          const { records } = await searchableStorage.find(COUNTRY, { searchKeys: '筆' }, {});
+          const { records } = await searchableStorage.find(COUNTRY, { searchKeys: 'aabb' }, {});
           await searchableStorage.batchDelete(COUNTRY, { recordKey: records.map((r) => r.recordKey) }).catch(noop);
```
Comment (artemgolozubov) [inline tests/integration/find-records-tests.ts:423]:
```suggestion
```

**Review by artemgolozubov — COMMENTED** [review]
Please remove lines introduced in integration tests in https://github.com/incountry/sdk-nodejs/pull/245 and https://github.com/incountry/sdk-nodejs/pull/247, it's no longer needed.

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by RommeroInCountry — APPROVED** [review]

**Pull request merged**